stock_deposit/views/stock_view.xml

```diff
@@ -0,0 +1,125 @@
+    <field name="arch" type="xml">
+        <xpath expr="//filter[@name='done']" position="after">
+            <!-- we need and operator -->
+            <separator/>
+            <filter name='deposit_loc'
+                    string="Deposit Locations"
+                    domain="[('location_dest_id.deposit_location', '=', True)]"/>
+            <filter name='deposit_loc_exclude'
+                    string="Deposit Locations Excluded"
+                    domain="[('location_dest_id.deposit_location', '!=', True)]"/>
+        </xpath>
+    </field>
+</record>
+
+<record id="deposit_act_product_stock_move_open" model="ir.actions.act_window">
+    <field name="context">{'search_default_product_id': active_id,
+                           'default_product_id': active_id
+                           'search_default_deposit_loc': 1}
+    </field>
+    <field name="name">Moves</field>
+    <field name="res_model">stock.move</field>
+    <field name="view_id" ref="stock.view_move_tree"/>
+</record>
+
+<record id="deposit_quantsact" model="ir.actions.act_window">
+    <field name="name">Deposit Quants</field>
+    <field name="res_model">stock.quant</field>
+    <field name="view_type">form</field>
+    <field name="view_mode">tree,form</field>
+    <field name="context">{'search_default_internal_loc': 1,
+                           'search_default_deposit_loc': 1,
+                           'group_by': ['product_id', 'location_id']}
+    </field>
+    <field name="help">
+        This analysis gives you a fast overview on the current stock level of
+        your products in deposit locations.
+    </field>
+</record>
+
+<menuitem id="menu_deposit_quants" name="Deposit Inventory Valuation"
+          parent="stock.menu_warehouse_report" sequence="125"
+          action="deposit_quantsact"/>
+
+<record id="deposit_view_picking_internal_search" model="ir.ui.view">
+    <field name="name">deposit.stock.picking.internal.search</field>
+    <field name="model">stock.picking</field>
+    <field name="inherit_id" ref="stock.view_picking_internal_search"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='partner_id']" position="after">
+            <separator/>
+            <filter name='deposit_loc'
+                    string="Deposit Locations"
+                    domain="[('location_dest_id.deposit_location', '=', True)]"/>
+        </xpath>
+    </field>
+</record>
+
+<record id="deposit_view_picking_form" model="ir.ui.view">
+    <field name="name">deposit.stock.picking.form</field>
+    <field name="model">stock.picking</field>
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='picking_type_id']" position="after">
+            <field name="is_deposit" invisible="1"/>
+        </xpath>
+        <xpath expr="//field[@name='owner_id']" position="attributes">
+            <attribute name="attrs">
+                {'required': [
```

Comment:
Don't make this format, as this makes the view unreadable in Odoo. Put it in one line.